packages/dds/sequence/src/test/fuzz/fuzzUtils.ts

```diff
@@ -303,6 +314,17 @@ export function createSharedStringGeneratorOperations(
 		};
 	}
 
+	async function annotateRange(state: ClientOpState): Promise<AnnotateRange> {
+		const { random } = state;
+		const key = random.pick(options.propertyNamePool);
+		const value = random.string(5);
```

Comment:
we also need to support the value `undefined` to cover removing annotations

Reply:
actually we should use both null and undefined as they have subtly different behavior